packages/components/src/components/hds/accordion/item/index.hbs

```diff
@@ -35,7 +35,7 @@
       @color="primary"
       id={{this.contentId}}
     >
-      {{yield to="content"}}
+      {{yield (hash close=c.close) to="content"}}
```

Comment:
just a consideration: the asymmetry here, that we have a `close` and not an `open` as well, does it mean something? are we missing something? 🤔

Reply:
if the content is `open` there's no need to provide a way to open the accordion; conversely, if the content is closed there is no element to perform an `open` function

Reply:
ah yes, this is yielded to the `::content` 🤦‍♂️